creme/stream/test_sql.py

```diff
@@ -1,4 +1,5 @@
-from sql import iter_sql # type: ignore
+import sql
+from sql import iter_sql 
```

Comment:
You need to do `from . import sql`. Then you can call `sql.iter_sql`.

Reply:
oh thanks god, the struggle was real

Reply:
Yeah importing modules when you're developping a library takes some practice. I suggest reading [this](http://www.blog.pythonlibrary.org/2016/03/01/python-101-all-about-imports/) article.